ONBOARD.md

```diff
@@ -0,0 +1,104 @@
+## Prepare configuration for component repo
+
+The JSON configuration file describes how to build different images in a
+testing pipeline for your repository. CI Operator has different *”targets”*
+that describe the goal images to build, and later targets build on successfully
+built earlier targets.
+
+### Source code image target
+
+By default, CI Operator builds the `src` target image. The `src` image is
```

Comment:
Use `ci-operator` everywhere instead of `CI Operator`.  It's a binary tool vs a more formal project.